apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -240,6 +243,13 @@ type ProjectCloneConfig struct {
 	Env []corev1.EnvVar `json:"env,omitempty"`
 }
 
+type ConfigmapReference struct {
```

Comment:
done

Reply:
Reverted back because the [types.NamespacedName](https://github.com/kubernetes/apimachinery/blob/c9c3e94f52f06444120a09bfbdb11bec38a5c8e1/pkg/types/namespacedname.go#L27-L30) fields do not have the json representation.

Reply:
@vinokurig sounds good - my mistake about that.